### PR TITLE
Cleanup product education and remove old promo strings

### PR DESCRIPTION
**Pull request by cklosowski**
This removes strings and logic around old promotional layouts. We should remove any translatable text we don't need any more so we can help translators out a bit.

### CODE REVIEW

File: includes/admin/discounts/contextual-help.php [hunk 1]
```diff
@@ -21,14 +23,17 @@
...
-		'<p>' . sprintf(
-			__( 'Need more from your Easy Digital Downloads store? <a href="%s">Upgrade Now</a>!', 'easy-digital-downloads' ),
-			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' )
-		) . '</p>'
-	);
+	$pass_manager = new Pass_Manager();
+	if ( $pass_manager->isFree() ) {
+		$screen->set_help_sidebar(
+			'<p><strong>' . __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
+			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://docs.easydigitaldownloads.com/' ) ) . '</p>' .
```
Comment (robincornett) [inline includes/admin/discounts/contextual-help.php:30]:
```suggestion
			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://easydigitaldownloads.com/docs/' ) ) . '</p>' .
```

File: includes/admin/downloads/contextual-help.php [hunk 2]
```diff
@@ -21,17 +23,21 @@
...
-			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' )
-		) . '</p>'
-	);
+	}
+
+	$pass_manager = new Pass_Manager();
+	if ( $pass_manager->isFree() ) {
+		$screen->set_help_sidebar(
+			'<p><strong>' . __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
+			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://docs.easydigitaldownloads.com/' ) ) . '</p>' .
```
Comment (robincornett) [inline includes/admin/downloads/contextual-help.php:34]:
```suggestion
			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://easydigitaldownloads.com/docs/' ) ) . '</p>' .
```

File: includes/admin/payments/contextual-help.php [hunk 3]
```diff
@@ -32,14 +34,17 @@ function edd_payments_contextual_help() {
...
-		'<p>' . sprintf(
-			__( 'Need more from your Easy Digital Downloads store? <a href="%s">Upgrade Now</a>!', 'easy-digital-downloads' ),
-			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' )
-		) . '</p>'
-	);
+	$pass_manager = new Pass_Manager();
+	if ( $pass_manager->isFree() ) {
+		$screen->set_help_sidebar(
+			'<p><strong>' . __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
+			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://docs.easydigitaldownloads.com/' ) ) . '</p>' .
```
Comment (robincornett) [inline includes/admin/payments/contextual-help.php:41]:
```suggestion
			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://easydigitaldownloads.com/docs/' ) ) . '</p>' .
```

File: includes/admin/reporting/contextual-help.php [hunk 4]
```diff
@@ -22,17 +24,21 @@
...
-		'<p>' . sprintf(
-			__( 'Need more from your Easy Digital Downloads store? <a href="%s">Upgrade Now</a>!', 'easy-digital-downloads' ),
-			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' )
-		) . '</p>'
-	);
+	$pass_manager = new Pass_Manager();
+	if ( $pass_manager->isFree() ) {
+		$screen->set_help_sidebar(
+			'<p><strong>' . __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
+			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://docs.easydigitaldownloads.com/' ) ) . '</p>' .
```
Comment (robincornett) [inline includes/admin/reporting/contextual-help.php:35]:
```suggestion
			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://easydigitaldownloads.com/docs/' ) ) . '</p>' .
```

File: includes/admin/settings/contextual-help.php [hunk 5]
```diff
@@ -26,14 +28,17 @@ function edd_settings_contextual_help() {
...
-		'<p>' . sprintf(
-			__( 'Need more from your Easy Digital Downloads store? <a href="%s">Upgrade Now!</a>.', 'easy-digital-downloads' ),
-			esc_url( 'https://easydigitaldownloads.com/pricing/?utm_source=plugin-settings-page&utm_medium=contextual-help-sidebar&utm_term=pricing&utm_campaign=ContextualHelp' )
-		) . '</p>'
-	);
+	$pass_manager = new Pass_Manager();
+	if ( $pass_manager->isFree() ) {
+		$screen->set_help_sidebar(
+			'<p><strong>' . __( 'For more information:', 'easy-digital-downloads' ) . '</strong></p>' .
+			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://docs.easydigitaldownloads.com/' ) ) . '</p>' .
```
Comment (robincornett) [inline includes/admin/settings/contextual-help.php:35]:
```suggestion
			'<p>' . sprintf( __( 'Visit the <a href="%s">documentation</a> on the Easy Digital Downloads website.', 'easy-digital-downloads' ), esc_url( 'https://easydigitaldownloads.com/docs/' ) ) . '</p>' .
```

**Review by robincornett — CHANGES_REQUESTED** [review]

**Comment by cklosowski** [comment]
@robincornett accepted all your changes for docs.

**Review by robincornett — APPROVED** [review]

**Pull request merged**